java/ChangeLog

```diff
@@ -0,0 +1,2 @@
+04/22/2017 Erez Makavy <hepi.yellow@gmail.com>
```

Comment:
If we're going to have a ChangeLog (and I'm personally undecided), can you add some info about v0.0.1 too please.